Detector/DetFCChhHCalTile/src/HCalBarrel_geo.cpp

```diff
@@ -105,10 +105,10 @@ static DD4hep::Geometry::Ref_t createHCal (
   double dy0 = moduleDimensions.dz();
   double dz0 = moduleDimensions.dr() / 2.;
 
-  double drWedge = (dimensions.rmax() - dimensions.rmin()) * 0.5;
+  double drWedge = cos(dphi / 2.) * (dimensions.rmax() - dimensions.rmin()) * 0.5;
```

Comment:
Hi Joschka,
I agree with the proposal. I think it is better to keep everything in the current envelope.
I was looking at the changes proposed for the FrontPlate and EndPlate. Seems like FrontPlate has a width of 0 at the moment. Valentine, could you please fix that? After that fix, we would need to correct barrelRmin,drWedge, numModulesR etc. to take into account that the active calorimeter starts at dimensions.rmin() + dRhoFacePlate. Also endPlateShape should start from here in r. Right?
Thanks,
Jana